handler/randomizer.go

```diff
@@ -0,0 +1,61 @@
+package handler
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+	k8upv1alpha1 "github.com/vshn/k8up/api/v1alpha1"
+	"math/big"
+)
+
+const (
+	ScheduleHourlyRandom   = "@hourly-random"
+	ScheduleDailyRandom    = "@daily-random"
+	ScheduleYearlyRandom   = "@yearly-random"
+	ScheduleAnnuallyRandom = "@annually-random"
+	ScheduleMonthlyRandom  = "@monthly-random"
+	ScheduleWeeklyRandom   = "@weekly-random"
+)
+
+func (s *ScheduleHandler) createSeed(schedule *k8upv1alpha1.Schedule, jobType k8upv1alpha1.JobType) string {
+	return schedule.Namespace + "/" + schedule.Name + "@" + string(jobType)
+}
+
+// randomizeSchedule randomizes the given originalSchedule with a seed. The originalSchedule has to be one of the supported
+// '@x-random' predefined schedules, otherwise it returns an error with the original schedule unmodified.
+func randomizeSchedule(seed, originalSchedule string) (string, error) {
+	checksum := calculateChecksumFromSeed(seed)
+
+	minute := new(big.Int).Set(checksum).Mod(checksum, big.NewInt(60))
```

Comment:
```suggestion
	minute := new(big.Int).Mod(checksum, big.NewInt(60))
```
The call to `Set` does effect the end result. Applies to all other 

It might be worth to extract that into a function.